src/NotificationEventMailing.php

```diff
@@ -271,20 +271,34 @@ public static function send(array $data)
                                 $doc->getFromDB($docID);
 
                                 //find width
-                                $width = null;
+                                $custom_width = null;
                                 if (preg_match("/width=[\"|'](\d+)(\.\d+)?[\"|']/", $matches[0][$pos], $wmatches)) {
-                                    $width = intval($wmatches[1]);
+                                    $custom_width = intval($wmatches[1]);
                                 }
-                                $height = null;
+                                $custom_height = null;
                                 if (preg_match("/height=[\"|'](\d+)(\.\d+)?[\"|']/", $matches[0][$pos], $hmatches)) {
-                                    $height = intval($hmatches[1]);
+                                    $custom_height = intval($hmatches[1]);
+                                }
+
+                                $img_infos  = getimagesize(GLPI_DOC_DIR . "/" . $doc->fields['filepath']);
```

Comment:
I do not know if that operation can be huge, but it may be conditioned; it does not seems used at all if both `custom` variable are not null.